CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 2.6)
 project(beatwave)
 
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++0x -Wall -Wextra -pedantic")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -Wall -Wextra -pedantic")
```

Comment:
Looks like Travis finally upgraded GCC to [4.8.4](https://travis-ci.org/rexim/beatwave/jobs/160094620#L308)! I used `c++0x` because when I just setup the CI it used [4.6.3](https://travis-ci.org/rexim/beatwave/jobs/147743351#L193) and `c++11` simply was [not supported](https://travis-ci.org/rexim/beatwave/jobs/147743351#L227). Looks like I can finally close #54. Thanks!